docs/css/app.css

```diff
@@ -27,7 +27,7 @@ body, input {
   font-family: cash-market, "Helvetica Neue", helvetica, sans-serif;
   line-height: normal;
   font-weight: bold;
-  color: #353535;
+  color: var(--md-default-fg-color)
```

Comment:
This change means the header colour dynamically changes between light/dark mode

Before: 

<img width="713" alt="image" src="https://user-images.githubusercontent.com/897017/200116630-b9381488-178f-4096-86ed-e99a33a0b743.png">

After:
<img width="634" alt="image" src="https://user-images.githubusercontent.com/897017/200116636-3c899054-8d50-418b-9d46-2f32bc362a71.png">